digdag-spi/src/main/java/io/digdag/spi/CommandStatus.java

```diff
@@ -0,0 +1,16 @@
+package io.digdag.spi;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+public interface CommandStatus
```

Comment:
Can we put some more documents on this API? Like:

getStatusCode is valid only when isFinished returns true
getIoDirectory returns the same String with CommandExecutorRequest.getIoDirectory

Reply:
Let me put on https://github.com/treasure-data/digdag/pull/835/commits/702e826057a70fe10d19d96990498ab1cb17de03